src/components/POLine/Cost/CostForm.js

```diff
@@ -101,6 +104,10 @@ class CostForm extends Component {
     const discountType = get(formValues, 'cost.discountType', DISCOUNT_TYPE.amount) || DISCOUNT_TYPE.amount;
     const isAmountDiscountType = discountType === DISCOUNT_TYPE.amount;
     const poLineEstimatedPrice = calculateEstimatedPrice(formValues);
+    const DEFAULT_CURRENCY_OPTIONS = [{ label: 'USD', value: 'USD' }];
```

Comment:
I think it should be moved out of render function

Reply:
Fixed